pkgs/development/compilers/emscripten/default.nix

```diff
@@ -42,7 +42,7 @@ stdenv.mkDerivation rec {
   patches = [
     (substituteAll {
       src = ./0001-emulate-clang-sysroot-include-logic.patch;
-      resourceDir = "${llvmEnv}/lib/clang/18/";
+      resourceDir = "${llvmEnv}/lib/clang/19/";
```

Comment:
You will want to use `lib.versions.major llvmEnv.version` instead of specifying the actual number.

Reply:
Of course. Thank you. Will adjust.

Reply:
Edited with latest force push.